Servers/UI/OJS.Servers.Ui/ClientApp/src/layout/search-bar/SearchBar.tsx

```diff
@@ -38,16 +38,28 @@ const SearchBar = () => {
     const [ searchParam, setSearchParam ] = useState<string>(defaultState.state.searchValue);
     const [ selectedTerms, setSelectedTerms ] = useState(defaultState.state.selectedTerms);
 
-    const { state: { isVisible }, actions: { toggleVisibility } } = useSearch();
+    const { state: { isVisible, getSearchResultsUrlParams }, actions: { toggleVisibility } } = useSearch();
```

Comment:
Separate the properties and action on new lines. We don't want to create horizontal rows.